### ISSUE TITLE
Hard to see the border of “Preview Changes” window when installing a package into a new project in Dark theme

### ISSUE THREAD

**Issue by v-longmin** [NuGet/Home#8275]
## Details about Problem
VS Version: D16.2stg 29027.113
OS Version: 17763.RS5_Release.180914-1434.
Installation:
Install VS: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=2806963&view=results
Install NuGet: \\\ddfiles\Drops\NuGet\Drops\CI\NuGet.Client\Dev\5.2.0.6067\artifacts\VS15\NuGet.Tools.vsix
## Detailed repro steps
1. Launch Visual Studio 2019 and select “Dark” as the theme.
2. Create a new C# Console App (.Net Core) project and open PM UI in Solution Explorer window.
3. Install a package (e.g. NUnit) into the project.
4. Observe “Preview Changes” window.
## Expected
The “Preview Changes” window can be distinguished by its border.
## Actual
Hard to see the border of “Preview Changes” window when installing a package into a new project in Black theme as below screenshot.
![image](https://user-images.githubusercontent.com/40230425/60335050-7d3de880-99cf-11e9-86ab-a7bae58bd728.png)
## Note
Similar bug 891275 has been fixed.

**Comment by nkolev92** [comment]
Duplicate of #

**Comment by v-longmin** [comment]
Verified the issue on NuGet client Dev\6.8.0.59, it has been fixed, close it.
